build/helper/helper.py

```diff
@@ -97,3 +98,27 @@ def get_numpy_type_for_api_type(api_type, config):
         # We didn't find it so assert
         assert False, 'Unknown value for api_type: {0}'.format(api_type)
 
+
+def get_array_type_for_api_type(api_type):
+    '''Returns the array type to use in the Python API from the original visa or custom type used in the C API
+
+    Do not use this with enums.
```

Comment:
can you assert this?

Reply:
No, not more than it already does. I don't have the information to know that this is an enum. It will end up asserting on the final one saying 'Unknown type'